dist/index.html

```diff
@@ -0,0 +1,137 @@
+          <button class="btn-resume">
+            <img src="img/topbar/download.png" alt="">
+          </button>
+          <p class="p-resume">DOWNLOAD RESUME</p>
+        </div>
+      </div>
+      <h1 class="greetengs">Hello, i’m <span class="span-name">John</span> <span class="span-surname">Smith</span></h1>
+      <p class="text-about">nec ullamcorper augue sagittis. Proin faucibus lorem nisl, Donec commodo leo est, quis eleifend urna pellentesque eget.</p>
+      <p class="text-about contact">Proin scelerisque odio eu nisi viverra, nec ullamcorper gue
+        <a href="mailto:contactme@mysite.com" class="contact-me">contact me</a></p>
+      <div class="skills">
+        <div class="skill-item photoshop">
+          <h4>PHOTOSHOP</h4>
+          <div class="progress">
+            <span class="span-skill" style="width: 98%; background: #c62fc6;"></span>
+          </div>
+          <p>98%</p>
+        </div>
+        <div class="skill-item illustrator">
+          <h4>ILLUSTRATOR</h4>
+          <div class="progress">
+            <span class="span-skill" style="width: 70%; background: #f2d25d;"></span>
+          </div>
+          <p>70%</p>
+        </div>
+        <div class="skill-item html-css">
+          <h4>HTML / CSS</h4>
+          <div class="progress">
+            <span class="span-skill" style="width: 60%; background: #2959e1;""></span>
+          </div>
+          <p>60%</p>
+        </div>
+      </div>
+      <div class="experience">
+        <div class="experience-item">
+          <h2>2000 - 2008</h2>
+          <p>Jr. Graphic Designer<span class="span-vertical-bar"> | </span><span class="span-company">ABC Inc.</span></p>
+        </div>
+        <div class="experience-item">
+          <h2>2008 - 2010</h2>
+          <p>Graphic Specialist<span class="span-vertical-bar"> | </span><span class="span-company">GRAPHICSTUDIO</span></p>
+        </div>
+        <div class="experience-item">
+          <h2>2010 - Present</h2>
+          <p>Creative Director<span class="span-vertical-bar"> | </span><span class="span-company">DESIGN ARTS</span></p>
+        </div>
+      </div>
+  </div>
+
+
+  <header id="showcase">
+    <div class="container">
+      <!-- Topbar -->
+      <div class="topbar">
+        <a href="#showcase" class="logo">
+          <img src="img/logo/logo.png" alt="">
+        </a>
+        <div class="burger">
+          <span></span>
+        </div>
+      </div>
+      
+      <!-- Title -->
+      <div class="showcase-text">
+        <h2>Hello,</h2>
+        <p class="p-secondary">Donec commodo leo est, quis eleifend urna pellentesque eget. Proin scelerisque odio eu nisi viverra, nec ullamcorper augue sagittis. Proin faucibus lorem nisl, </p>
+        <img src="img/graph_shapes/shape1.png" alt="" class="shape1">
+      </div>
+    </div>
+  </header>
+
+  <!-- Main -->
+  <section class="section-main">
+    <div class="container overflow">
+      <div class="works">
+        <div class="work blue">
```

Comment:
don't create classes with name of color. Color could be changed in design for a few seconds, but in your case it will take a time to change it in html and css